src/Components/LoginForm.tsx

```diff
@@ -0,0 +1,228 @@
+import React, {
+  ChangeEvent,
+  FormEvent,
+  useContext,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import classNames from 'classnames';
+import { AppContext } from '../AppContext';
+import { findUser, addUsers } from '../api/todos';
+import { Notification } from './Notification';
+
+const addPrevUser = (prevEmail: string) => {
+  const getPrevUsers: string[] | 0 = JSON
+    .parse(localStorage.getItem('prevusers') || '0');
```

Comment:
not fixed
<img width="490" alt="Screenshot 2023-04-08 at 20 38 47" src="https://user-images.githubusercontent.com/32853034/230737616-fd8b2276-e8ea-42e1-b818-40a48a700d2f.png">